tests/foreman/cli/test_subscription.py

```diff
@@ -244,479 +228,3 @@ def test_negative_manifest_refresh(self):
-                'organization-id': self.org['id'],
-                'id': activation_key['id'],
-            }, output_format='json')
-            self.assertNotIn(
-                DEFAULT_SUBSCRIPTION_NAME, [sub['name'] for sub in ak_subs])
-            for host in hosts:
-                host_subscriptions = ActivationKey.subscriptions({
-                    'organization-id': self.org['id'],
-                    'id': activation_key['id'],
-                    'host-id': host['id'],
-                }, output_format='json')
-                self.assertNotIn(
-                    DEFAULT_SUBSCRIPTION_NAME,
-                    [sub['name'] for sub in host_subscriptions]
-                )
-            # upload the manifest again
-            self._upload_manifest(self.org['id'])
-            # ensure that the subscription was not auto attached
-            ak_subs = ActivationKey.subscriptions({
-                'organization-id': self.org['id'],
-                'id': activation_key['id'],
-            }, output_format='json')
-            self.assertNotIn(
-                DEFAULT_SUBSCRIPTION_NAME, [sub['name'] for sub in ak_subs])
-            for host in hosts:
-                host_subscriptions = ActivationKey.subscriptions({
-                    'organization-id': self.org['id'],
-                    'id': activation_key['id'],
-                    'host-id': host['id'],
-                }, output_format='json')
-                self.assertNotIn(
-                    DEFAULT_SUBSCRIPTION_NAME,
-                    [sub['name'] for sub in host_subscriptions]
-                )
-            # restore from the saved activation key and content hosts
-            # subscriptions
-            CSV_.activation_keys({
-                'file': ak_file_path,
-                'organization': self.org['name'],
-                'itemized-subscriptions': True,
-            })
-            CSV_.content_hosts({
-                'file': ch_file_path,
-                'organization': self.org['name'],
-                'itemized-subscriptions': True,
-            })
-            # ensure that the subscriptions has been restored
-            ak_subs = ActivationKey.subscriptions({
-                'organization-id': self.org['id'],
-                'id': activation_key['id'],
-            }, output_format='json')
-            self.assertIn(
-                DEFAULT_SUBSCRIPTION_NAME, [sub['name'] for sub in ak_subs])
-            for host in hosts:
-                host_subscriptions = ActivationKey.subscriptions({
-                    'organization-id': self.org['id'],
-                    'id': activation_key['id'],
-                    'host-id': host['id'],
-                }, output_format='json')
-                self.assertIn(
-                    DEFAULT_SUBSCRIPTION_NAME,
-                    [sub['name'] for sub in host_subscriptions]
-                )
-
-    @tier4
-    def test_positive_restore_content_hosts_with_modified_subscription(self):
```

Comment:
same as above and all the below removed tests ?

Reply:
same as above.